specs/www.rst

```diff
@@ -419,6 +428,15 @@ keys may be present, however.
 Disconnect - ``receive`` event
 ''''''''''''''''''''''''''''''
 
+.. note::
```

Comment:
For websockets the event had additional information, namely the `code`. Should we specify that the exception MUST have a `code` attribute?